.travis.yml

```diff
@@ -27,3 +31,26 @@ before_script:
 - psql -d c2cgeoform_demo_tests -c "CREATE EXTENSION postgis;" -U postgres
 
 script: make check && make test
+
+before_deploy:
+- make compile-catalog
```

Comment:
Not sure this is required as it is done by make build, and translations should be tested.

Reply:
But Travis clean the build result before running the deploy ...

Reply:
Ok thanks